Correction about smoothing the graph

dewi-ny-je opened the pull request:
It was incorrectly stated that only the marks were removed, but, even if I honestly didn't check the sources, it cannot be: with two points I can get normal graphs with marks, but I get an error when I enable smoothing: "at least three points are required for splines".

tsuna commented:
Can you make the change to `docs/source/user_guide/guis/index.rst` and then run `make` to rebuild the HTML?  If you change the HTML directly, your change will be lost next time someone does `make`.

dewi-ny-je commented:
Well I modified that file online, without cloning on my computer, so I couldn't do the make.
Now I cloned it, I modified the file you specified, then I run "make" but I got:
~/temp_repo/opentsdb.net $ make
make: Nothing to be done for `all'.

If I perform the "make" in the docs subdir, I get "The 'sphinx-build' command was not found." and so on.

Should I trick the system by placing the updated index.html in the correct position before the commit, without performing "make"? It's a small change anyway.

manolama commented:
Fixed up and merged in 6f9d527e24e514137024a6742a0ef1646924a579. You can hack around but it's easier just to install Sphinx :)
